cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/src/bond0.nmconnection

```diff
@@ -0,0 +1,17 @@
+[connection]
```

Comment:
Connection profile of the main bond0 interface

Reply:
Was this a question, or a suggestion for documentation, or....?